spec/serializers/finance/npq/assurance_report/csv_serializer_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Finance::NPQ::AssuranceReport::CsvSerializer do
```

Comment:
💯 for adding the missing spec; is it worth doing a bit of house keeping and adding coverage for the rest of the serialisation while you're in here?